robottelo/hosts.py

```diff
@@ -1402,6 +1411,24 @@ def register_to_dogfood(self):
                 f'Error during registration, command output: {cmd_result.stdout}'
             )
 
+    def register_to_cdn(self):
```

Comment:
This should go under the ContentHost class and raise a ContentHostError instead of CapsuleHostError

Reply:
@JacobCallahan I agree, ContentHost can also utilize this, updated, please check again, thanks.